packages/faustwp-cli/tests/healthCheck/validateFaustEnvVars.test.ts

```diff
@@ -50,3 +53,27 @@ describe('healthCheck/validateFaustEnvVars', () => {
     expect(mockExit).toBeCalledTimes(0);
   });
 });
+
+describe('isWPEngineComTLD', () => {
+  it('matches subdomains on wpengine.com', () => {
+    expect(isWPEngineComSubdomain('https://my-site.wpengine.com')).toBeTruthy();
+    expect(
+      isWPEngineComSubdomain('http://some-site.wpengine.com/graphql'),
+    ).toBeTruthy();
+    expect(isWPEngineComSubdomain('https://example.wpengine.com')).toBeTruthy();
+    expect(
+      isWPEngineComSubdomain('https://some-long-weird-subdomain.wpengine.com'),
+    );
+  });
+
+  it('does not match urls that are not subdomains of wpengine.com', () => {
+    expect(isWPEngineComSubdomain('https://example.com')).toBeFalsy();
+    expect(isWPEngineComSubdomain('https://wpengine.com')).toBeFalsy();
+    expect(isWPEngineComSubdomain('https://wpengine.com/plans')).toBeFalsy();
+    expect(isWPEngineComSubdomain('https://my-site.wpengine.co')).toBeFalsy();
+    expect(
+      isWPEngineComSubdomain('https://my-site.wpenginepowered.com'),
+    ).toBeFalsy();
+    expect(isWPEngineComSubdomain('https://my-site.wpengine.co'));
```

Comment:
This looks like a partial dupe of line 73.

```suggestion
```

Reply:
@mindctrl Good eye. thanks!